src/WP_CLI/Fetchers/Signup.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace WP_CLI\Fetchers;
+
+/**
+ * Fetch a signup based on one of its attributes.
+ */
+class Signup extends Base {
```

Comment:
Should this fetcher live within https://github.com/wp-cli/wp-cli?

Reply:
@danielbachhuber Yah, I agree. I have already mentioned about it above comment. https://github.com/wp-cli/entity-command/pull/489#issuecomment-2006025162 But I am not sure how to proceed about that. Should that class be merged in main repo and then continue updating here in this PR? If class is ok, then I can create PR in main repo.